library/src/test/java/com/github/pwittchen/reactivenetwork/library/internet/observing/strategy/WalledGardenInternetObservingStrategyTest.java

```diff
@@ -26,21 +45,36 @@
   private static final int INTERVAL_IN_MS = 2000;
   private static final int PORT = 80;
   private static final int TIMEOUT_IN_MS = 30;
+  private static final String HOST_WITH_HTTP = "http://www.website.com";
+  private static final String HOST_WITH_HTTPS = "https://www.website.com";
+  private static final String HOST_WITHOUT_HTTPS = "www.website.com";
+
   @Rule public MockitoRule rule = MockitoJUnit.rule();
-  @Spy private WalledGardenInternetObservingStrategy strategy;
-  @Mock private ErrorHandler errorHandler;
+  @Spy public WalledGardenInternetObservingStrategy strategy;
+  @Spy public ErrorHandler errorHandler = new ErrorHandler() {
+    @Override
+    public void handleError(Exception exception, String message) {
+      // do nothing
+    }
+  };
+
+  @Before
+  public void setUp() throws Exception {
+    Mockito.doNothing().when(errorHandler).handleError(ArgumentMatchers.any(Exception.class), ArgumentMatchers.anyString());
+  }
 
   private String getHost() {
     return strategy.getDefaultPingHost();
   }
 
   @Test public void shouldBeConnectedToTheInternet() {
     // given
-    when(strategy.isConnected(getHost(), PORT, TIMEOUT_IN_MS, errorHandler)).thenReturn(true);
+    String host = getHost();
+    Mockito.doReturn(true).when(strategy).isConnected(host, PORT, TIMEOUT_IN_MS, errorHandler);
```

Comment:
Please, use static import to exclude `Mockito` class name. It should look like: `doReturn(true).when(...)`. All tests in this project use that convention.